pyseir/inference/model_fitter.py

```diff
@@ -687,31 +714,16 @@ def run_state(state, states_only=False):
     state_obj = us.states.lookup(state)
     logging.info(f'Running MLE fitter for state {state_obj.name}')
 
-    model_fitter = ModelFitter.run_for_fips(state_obj.fips)
-
-    df_whitelist = load_data.load_whitelist()
-    df_whitelist = df_whitelist[df_whitelist['inference_ok'] == True]
-
-    output_path = get_run_artifact_path(state_obj.fips, RunArtifact.MLE_FIT_RESULT)
-    pd.DataFrame(model_fitter.fit_results, index=[state_obj.fips]).to_json(output_path)
-
-    with open(get_run_artifact_path(state_obj.fips, RunArtifact.MLE_FIT_MODEL), 'wb') as f:
-        pickle.dump(model_fitter.mle_model, f)
+    _execute_model_for_fips(_execute_model_for_fips(state_obj.fips))
 
     # Run the counties.
     if not states_only:
+        df_whitelist = load_data.load_whitelist()
+        df_whitelist = df_whitelist[df_whitelist['inference_ok'] == True]
+
         all_fips = df_whitelist[df_whitelist['state'].str.lower() == state_obj.name.lower()].fips.values
 
         if len(all_fips) > 0:
             p = Pool()
-            fitters = p.map(ModelFitter.run_for_fips, all_fips)
+            p.map(_execute_model_for_fips, all_fips)
             p.close()
-
-            county_output_file = get_run_artifact_path(all_fips[0], RunArtifact.MLE_FIT_RESULT)
-            pd.DataFrame([fit.fit_results for fit in fitters if fit]).to_json(county_output_file)
```

Comment:
here's the issue

Reply:
What do you mean by issue?

Reply:
it was a note to myself when i was working through the logic